justfile

```diff
@@ -0,0 +1,13 @@
+# Justfile to make some common tasks in the vodozemac repo easier.
+
+# List the available recipes.
+default:
+    just --list
+
+# Create a coverage report using llvm-cov.
+coverage:
+    cargo llvm-cov nextest
+
+# Create a coverage report for codecov.
```

Comment:
> # Create a coverage report using llvm-cov.
> # Create a coverage report for codecov.

one of these says which tool it uses to *make* the report (`llvm-cov`) while the other says which tool the report is *for* (`codecov`). Don't they both use `llvm-cov`? If the report generated by `coverage` is not for `codecov`, what is it for?

Reply:
Or drop the "using llvm-cov" from both if you think it's unimportant

Reply:
I think it's fine to mention it, as the tool has many more options. The justfile is here to more easily use the default flows and aid in the discoverability of the coverage report support we have.